cdap-watchdog/src/main/java/co/cask/cdap/logging/save/KafkaMessageCallback.java

```diff
@@ -94,7 +95,7 @@ public void onReceived(Iterator<FetchedMessage> messages) {
       } catch (Throwable th) {
         LOG.error("Error processing message at topic {} parition {}",
```

Comment:
If logging the message is too large, it would be good to log the message offset so that it can be debug later (as long as it's still in Kafka).